### ISSUE TITLE
Hacked Cargo console does nothing

### ISSUE THREAD

**Issue by TheOmn1** [OracleStation/OracleStation#740]
Note: This is distinctly different from an emagged cargo console.

For those who do not know what I am talking about, you can deconstruct a cargo console until you have removed its board. You can then multitool the board (change its spectrum to Broad) and then reinstall it into the console. The newly rebuilt cargo console should have an expanded list of items which you can order.

I've tried to redo this process again (when the menu of items which can be ordered has not changed) and have found multitooling the same board to ask me if I want to change to Broad again. This suggests the board sorts of resets after being reinstalled (because on the initial point of changing it to broad, multitooling it again will ask me if I want to make it standard) I don't know the actual problem so this is only an assumption.

**Comment by FlattestGuitar** [comment]
I can't reproduce this. Items show up as expected when the thing is hit with a multitool. 

Stuff like the Biker Gang Kit or the High-yield Clown-grade Cream Pie Crate are all there. Gonna leave this open in case someone can shed any light on why this happened to you.

**Comment by TheOmn1** [comment]
I tried it again in round 1550 as a cargo tech.

Disassembled the console to remove the board, multitooled the board, reinserted and reinstalled the entire thing.

Still same situation. No expanded menu of items despite bring broad.

I tried to multitool the console at each step just in case that did the trick, maybe there is something different in the server and you suggested hitting it with a multitool, still no difference.
